app/src/Containers/SearchContainer.tsx

```diff
@@ -241,31 +240,35 @@ const SearchContainer = () => {
     return undefined
   }
 
+  
   const handleQuickSearchSubmit = (
     searchField: 'client' | 'description' | 'name' | 'role' | 'skills' | 'sector',
     searchQueryChoice: string,
   ) => {
     let searchQueries = [] as string[]
+    let results = [] as Projects
+    
+    searchQueries = searchQueries.concat(searchQuery)
```

Comment:
Should this be 
```javascript
searchQueries = searchQueries.concat(searchQueryChoice)
```
?

Reply:
No, this is the line causing the errors @M4RC1N76 has caught in testing. I did test but must have had a cache kick in or something (also bit tired last night). All resolved now. 

Thanks for catching this!